src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -0,0 +1,74 @@
+        self._parser = parser
+        self._add_deprecated_keywords_to_parser()
+
+    REPLACE_WITH_GEN_KW = ["RELPERM", "MULTZ", "EQUIL", "GEN_PARAM"]
+
+    def _add_deprecated_keywords_to_parser(self):
+        for kw in self.REPLACE_WITH_GEN_KW:
+            self._parser.add(kw)
+        self._parser.add("UMASK")
+        self._parser.add("HAVANA_FAULT")
+        self._parser.add("MULTFLT")
+        self._parser.add("REFCASE_LIST")
+        self._parser.add("RFTPATH")
+        self._parser.add("END_DATE")
+
+    def suggest_migrations(self, filename: str):
+        suggestions = []
+
+        content = self._parser.parse(filename)
+        for kw in self.REPLACE_WITH_GEN_KW:
+            if content.hasKey(kw):
+                suggestions.append(
+                    "The keyword {kw} was deprecated in 2009 in favor of using"
+                    " GEN_KW and FORWARD_MODEL."
+                )
+        if content.hasKey("UMASK"):
+            suggestions.append(
+                f"The UMASK keyword no longer has any effect "
+                "and can safely be removed from the config file."
+            )
+        if content.hasKey("HAVANA_FAULT"):
+            suggestions.append(
+                "Direct interoperability with havana was removed from ert in 2009."
+                " The behavior of HAVANA_FAULT can be reproduced using"
+                " GEN_KW and FORWARD_MODEL."
+            )
+        if content.hasKey("MULTFLT"):
+            suggestions.append(
+                "The MULTFLT keyword was replaced by the GENKW keyword in 2009."
+                "Please see https://ert.readthedocs.io/en/latest/"
+                "reference/configuration/keywords.html#gen-kw"
+                "to see how to migrate from MULTFLT to GEN_KW."
+            )
+        if content.hasKey("REFCASE_LIST"):
+            suggestions.append(
+                "The REFCASE_LIST keyword was used to give a .DATA file "
+                "to be used for plotting. The corresponding plotting functionality "
+                "was removed in 2015, and this keyword can be safely removed from "
+                "the config file."
+            )
+        if content.hasKey("RFTPATH"):
+            suggestions.append(
+                "The RFTPATH keyword was used to give a path to well observations "
+                "to be used for plotting. The corresponding plotting functionality "
+                "was removed in 2015, and the RFTPATH keyword can safely be removed "
+                "from the config file."
+            )
+        if content.hasKey("END_DATE"):
+            suggestions.append(
+                "The END_DATE keyword was used to check that a the dates in a summary "
+                "file would go beyond a certaind date. This would only display a "
+                "warning in case of problems. The keyword has since been deprecated, "
+                "and can be safely removed from the config file."
+            )
```

Comment:
Consider adding info for:

```
Deprecated:
SCHEDULE_PREDICTION_FILE
https://ert.readthedocs.io/en/latest/reference/configuration/keywords.html#schedule-prediction-file
CASE_TABLE
https://ert.readthedocs.io/en/latest/reference/configuration/keywords.html#case-table
RERUN_START
https://ert.readthedocs.io/en/latest/reference/configuration/keywords.html#rerun-start

Ignored:
LOG_FILE
https://ert.readthedocs.io/en/latest/reference/configuration/keywords.html#log-file
LOG_LEVEL
https://ert.readthedocs.io/en/latest/reference/configuration/keywords.html#log-level

Breaking:
PORTABLE_EXE into EXECUTABLE

Removed:
DELETE_RUNPATH
SINGLE_NODE_UPDATE
GEN_PARAM
RSH (queue driver)
RSH_HOST
RSH_COMMAND
BINARY_FLOAT
BINARY_DOUBLE

Reply:
I worked a bit more to figure out what more I can add. I believe adding RSH* should be fairly straight forward as the simple matter is that we have deprecated the `RSH_DRIVER`. I also attempted to figure out anything about the `RERUN_START` but I couldn't figure out much except that the functionality got lost sometime between 2016 and 2017.

For `PORTABLE_EXE` we need to look in job configs, so I suggest we do not implement that in this PR.

Reply:
I found out what `RERUN_START` was all about, and I have added that as well.

Reply:
So the whole `DELETE_RUNPATH` thing was removed back in 2017, and what was left and we just recently removed was just the docs. But lets add it to the suggester for giggles.

Reply:
Great work diving for details in the history logs. :+1: